src/unifiedreport/ui/agent-foreport.php

```diff
@@ -49,7 +49,7 @@ protected function handle(Request $request)
     $userId = Auth::getUserId();
     $jobId = JobAddJob($userId, $groupId, $upload->getFilename(), $uploadId);
     $error = "";
-    $jobQueueId = $reportGenAgent->AgentAdd($jobId, $uploadId, $error, array());
+    $jobQueueId = $reportGenAgent->AgentAdd($jobId, $uploadId, $error, array(), tracebackTotalUri());
```

Comment:
I am a little bit confused why you are storing the URI in the `jq_cmd_args`. At first sight, it is not clear to me, why the URI should be a "command argument".  Or is `jq_cmd_args` intended to be a general field for remarks about a job?  Anyways, its it really necessary to store this (redundant) information in the database? Is there a reason why you don't simply call `tracebackTotalUri()` in `UnifiedReport`?

Reply:
As the report gets generated in agent side i have to pass it through command line args. 

tracebackTotalUri() only works for UI not CLI.